apps/prairielearn/elements/pl-string-input/pl-string-input.py

```
@@ -227,6 +251,9 @@ def parse(element_html: str, data: pl.QuestionData) -> None:
     if remove_spaces:
         a_sub = "".join(a_sub.split())
 
+    # Always simplify multiline characters (if they still exist)
+    a_sub = re.sub("\r*\n", "\n", a_sub)
```

Ruff (0.8.2)

First argument to `re.sub()` is not raw string

(RUF039)